src/aiida_quantumespresso/cli/utils/validate.py

```diff
@@ -59,9 +59,9 @@ def validate_hubbard_parameters(structure, parameters, hubbard_u=None, hubbard_v
     if hubbard_file_pk:
 
         try:
-            hubbard_file = load_node(pk=hubbard_file_pk)
-        except exceptions.NotExistent as exc:
-            raise ValueError(f'{hubbard_file_pk} is not a valid pk') from exc
+            hubbard_file = load_node(pk=hubbard_file_pk.pk)
```

Comment:
Actually, I am not sure why here the `hubbard_file_pk` is loaded instead of being passed as an integer. At least, that what it seems happening in the tests.

Reply:
Because that value comes from the `hubbard_file_pk` argument in the CLI command which is provided by the `HUBBARD_FILE` reusable option. That option takes an identifier on the command line (pk, uuid or label), but it _returns_ the corresponding node, already loaded. I would recommend changing the variable name of that `HUBBARD_FILE` option from `hubbard_file_pk` to `hubbard_file` and in the `validate_hubbard_parameters` as well. The whole conditional `if hubbard_file_pk` can be removed, because it will already pass a node in that is guaranteed to be a `SinglelfileNode. Also update the docs of this method that `hubbard_file` is a `SinglefileData` with Hubbard parameters and not the pk of one.